draco/synthesis/expand_perturbed.py

```diff
@@ -0,0 +1,598 @@
+
```

Comment:
Richard gave me the advice to use 'flake8' with atom for style errors. 
There seem to be quite a lot of style errors in this file!